frontend/src/Pages/ResidentProfile.tsx

```diff
@@ -114,263 +140,242 @@ const ResidentProfile = () => {
+                                    metrics?.user.created_at
+                                        ? new Date(
+                                              metrics.activity_engagement.joined
+                                          ).toLocaleDateString('en-US', {
+                                              year: 'numeric',
+                                              month: 'short',
+                                              day: 'numeric'
+                                          })
+                                        : 'No Date Available'
                                 }
-                                label={avgLabel}
-                                tooltip="Average time spent in UnlockEd per week"
-                                useToLocaleString={false}
                             />
-                            <StatsCard
-                                title="Total Time This Week"
-                                number={
-                                    parseFloat(weekNumber) === 0
-                                        ? '0'
-                                        : parseFloat(weekNumber) < 1
-                                          ? '<1'
-                                          : weekNumber
+                            <UserProfileInfoRow
+                                column="Last Active"
+                                value={
+                                    metrics.activity_engagement.last_active_date
+                                        ? new Date().toLocaleDateString(
+                                              'en-US',
+                                              {
+                                                  year: 'numeric',
+                                                  month: 'short',
+                                                  day: 'numeric'
+                                              }
+                                          )
+                                        : 'N/A'
                                 }
-                                label={weekLabel}
-                                tooltip="Total time spent in UnlockEd this week"
                             />
+                            <div className="flex flex-row gap-2 mt-4">
+                                <button className="button bg-grey-1 text-error">
+                                    Delete Resident
+                                </button>
+                                {user && canSwitchFacility(user) && (
+                                    <button
+                                        className="button bg-grey-1 text-teal-4"
+                                        onClick={() =>
+                                            showModal(verifyResidentModal)
+                                        }
+                                    >
+                                        Transfer Resident
+                                    </button>
```

Comment:
Can you center these buttons? When the name is longer and stretches the width of the container the buttons are shifted more toward the left.